packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -69,4 +70,19 @@ governing permissions and limitations under the License.
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
+  outline: none;
+}
+
+.spectrum-Tag-action {
+  grid-area: action;
+  height: 75%;
```

Comment:
where'd 75% come from?

Reply:
I eyeballed it. When set to 100% the 'x' for remove functionality doesn't align with the text

Reply:
o, I see, I think we'd be better off setting height to `var(--spectrum-tag-height)`, this will appropriately size with large scale then

Reply:
you'd remove the padding top as well to do that, and you might need to subtract the border width from it as well
```
calc(var(--spectrum-tag-height) - (2 * var(--spectrum-tag-border-size)));
```

Reply:
and we'll want the width to be bigger, right now if you click just to the right of the 'x', it actually won't fire an onRemove, to do that, when there is a remove button, we'll want to drop the padding-inline-end to 0 i think

Reply:
![image](https://user-images.githubusercontent.com/20674023/140381484-66aa48c5-49b0-4ac0-a0aa-39f574e45eed.png)
I'm not the biggest fan of the little gap on the right between the tag end and the 'x'. i'm thinking we can pull the action flush against the tag. thoughts? @snowystinger

Reply:
Yeah, i think that'd be fine